test/runner.js

```diff
@@ -0,0 +1,192 @@
+'use strict'
+
+let fs = require('fs')
+let path = require('path')
+let expect = require('chai').expect
+let rimraf = require('rimraf')
+let Launcher = require('webdriverio/build/lib/launcher')
+let parseXmlString = require('xml2js').parseString
+
+let configFile = './test/fixtures/wdio.conf.js'
+let resultsDir = path.join(__dirname, '../allure-results') 
+
+describe('"before all" hook', () => {
+
+  beforeEach(() => {
+    clean()
+  })
```

Comment:
now your promise from `clean()` goes to nothing and Mocha doesn't wait on it. I'd recommend here to do this:

```js
beforeEach(clean)
```